AllContractsHashes.json

```diff
@@ -1806,173 +1094,5 @@
     "evmDeployedBytecodeHash": "0xbcf2d0de342544f95c33cddbc122cd29455067219c60555a59ee680745879fa7",
     "zkBytecodeHash": null,
     "zkBytecodePath": null
-  },
-  {
-    "contractName": "da-contracts/AvailL1DAValidator",
```

Comment:
please compile do:

`yarn sc build:foundry`
`yarn da build:fondry`

and then try to rerun yarn calculate hashes

Reply:
and also `yarn l2 build:foundry`